### PR TITLE
Flush stdout/err in loop to populate log file in real time

### PR DESCRIPTION
**Pull request by jhallard**

### CODE REVIEW

**Comment by jhallard** [comment]
simple changes, would be nice to get in ASAP so I can continue working off master.

File: box_api/register_batch_source.sh [hunk 1]
```diff
@@ -9,7 +9,7 @@
 # NOTE - this script assumed that you have some environment variables set, these varibles are
 # CAMIOTOKENPROD - access token (oauth) for you Camio account on the production servers
 # CAMIOIDPROD - ID of your camio account on production servers
-# then the same thing but for our testing server (test.camio.com)
+# then the same thing but for our tesing server (test.camio.com)
```
Comment (jcmaslan) [inline box_api/register_batch_source.sh]: testing

**Review by jcmaslan — APPROVED** [review]
minor typo to correct

**Comment by mdipierro** [comment]
LGTM

**Comment by jhallard** [comment]
@jcmaslan requested changes are in

**Review by jcmaslan — APPROVED** [review]

**Pull request merged**